frontend/src/pages/home/AlternatingBackgroundColorSectionList.tsx

```diff
@@ -2,45 +2,105 @@ import React from 'react';
 
 import { Grid, useTheme } from '@mui/material';
 
+const SectionGridItem = ({
+  child,
+  color,
+  background,
+  verticalPadding,
+}: {
+  child: React.ReactNode;
+  color: string;
+  background: string;
+  verticalPadding: string;
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Grid
+      item
+      xs={12}
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+        padding: verticalPadding,
+        [theme.breakpoints.down('md')]: {
+          padding: `${verticalPadding} 16px ${verticalPadding} 16px`,
+        },
```

Comment:
You can also write:

`
{
p: verticalPadding,
px: {xs: '16px', md: verticalPadding} 
}`

I don't know if verticalPadding is a right naming because it defined also horizontal padding if you display the desktop version

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1183/commits/41025b059d24bda944d1fcfd6f8cb837d6ab35f8